twarep/electrifly
# Updated Prescription Model Data and Statistical Calculative Procedures

## Checklist

## What I Did
- Updated the data input to the model by adding ground speed and pressure altitude.
- Saved all models to their specific `joblib` files.
- Nayeema added the following statistical error calculations mean squared error, cross-validated R-squared, and more.

## How I Did it?
- Used Python and SQL querying

## How to Test?
- No real need to test, you can run the python notebook files for yourself and see, just comment out the last code chunk in all files.
- No need to run the file `Prescription_data_engineering.ipynb` because all it does is creates the `all_date.csv` file.

### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
